bundle-linux.sh

```diff
@@ -72,7 +72,7 @@ bundleExe() {
   local copied_exe="$out/$exe_dir/$exe_name"
   cp "$exe" "$copied_exe"
   chmod +w "$copied_exe"
-  patchelf --set-interpreter "$(basename "interpreter")" --set-rpath "\$ORIGIN/../$lib_dir" "$copied_exe"
+  patchelf --set-interpreter "interpreter" --set-rpath "\$ORIGIN/../$lib_dir" "$copied_exe"
```

Comment:
The purpose of setting it to `"interpreter"` is just to make it not runnable on it's own, but require the bin script to provide the correct interpreter, right?

Reply:
We're not actually using this Interpreter, correct. The script overrides this.

Interpreter is likely a nix path here. Nuke refs would turn that path into a broken nix path. But I wanted to break it without making it obvious the file was built from nix.